lib/bcu.rb

```diff
@@ -48,17 +48,34 @@ def self.process(args)
 
     ohai "Found outdated apps"
     Formatter.print_app_table(outdated, state_info, options)
+    printf "\n"
 
-    if options.dry_run
-      printf "\nDo you want to upgrade %d app%s [y/N]? ", outdated.length, (outdated.length > 1) ? "s" : ""
+    unless options.interactive || options.force_yes
+      printf "Do you want to upgrade %d app%s [y/i/N]? ", outdated.length, (outdated.length > 1) ? "s" : ""
```

Comment:
maybe explain what `i` means here:
`"Do you want to upgrade all %d app%s or enter [i]nteractive mode [y/i/N]? "`

Reply:
Thx, updated